plugin.xml

```diff
@@ -93,7 +93,12 @@
         </config-file>
 
         <config-file target="AndroidManifest.xml" parent="/manifest">
-			<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+            <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
@zookz Sorry for the very late review and thank you for the contribution! Could you please explain why the `READ_EXTERNAL_STORAGE` permission is needed? 

Same question for `BLUETOOTH_ADMIN`. I haven't worked with beacons in a long time but these look like they might not be necessary for BLE monitoring/ranging. If you can provide sources showing that these are indeed necessary then I'm OK to leave this in as well.